### DISCUSSION TITLE
Reading the SyGuS file or string with Python API.

### DISCUSSION THREAD

**Discussion by pauljurczak** [cvc5/cvc5#9830] [Q&A] [answered]
The CLI version of cvc5 can read and solve SyGuS file. Is there a Python API to read and solve SyGuS file or string?

**Comment by ajreynol** [comment]
We are working on functionality for doing this.  For now, the input parser is available only in cpp as part of an unofficial API here: https://github.com/cvc5/cvc5/blob/main/src/parser/api/cpp/input_parser.h

We will keep you posted when this is finalized and available in Python.

**Reply by pauljurczak** [reply]
Thank you.

**Comment by ajreynol** [comment, marked as answer]
This is now officially in the API since 1.1.0.

**Comment by pauljurczak** [comment]
Thank you.
